code/test/sanityCheck.py

```diff
@@ -106,11 +107,28 @@ def checkUnusedEntities(model, entity_type):
     assert len(unused_entities) == 0, f"Found unused {entity_type}: {unused_entities}"
 
 
+def checkDupRxn(model):
+    """
+    Check for duplicate reactions in the model
+    """
+
+    reaction_equations = [rxn.build_reaction_string(use_metabolite_names=False) for rxn in model.reactions]
```

Comment:
I suspect that this would only catch duplicated reactions where the metabolites have the exact same order?

Reply:
I assume metabolites are often sorted, so that duplicated reactions can be detected

Reply:
the output might be further optimized by showing duplicate pairs in groups, instead of a list of rxn ids